File: OCaml/start-code/startcode.ml
(* Beschreibung: Ein einfaches OCaml-Programm *)
(* Datum: 25.01.2025 *)

(* Begrüßung *)
print_endline "Willkommen zu meinem OCaml-Programm!";;

(* Funktion definieren *)
let begruessung () =
  print_endline "Das ist eine Beispiel-Funktion in OCaml."
;;

(* Funktion aufrufen *)
begruessung ();;

(* Nutzereingabe *)
print_endline "Bitte geben Sie Ihren Namen ein: ";;
let name = read_line ();;

(* Begrüßung mit dem Namen *)
print_endline ("Hallo, " ^ name ^ "! Schön, dass Sie hier sind.");;

(* Programm beenden *)
print_endline "Das Programm wurde beendet."

File: Swift/start-code/startcode.swift
// Beschreibung: Ein einfaches Swift-Programm
// Datum: 25.01.2025

import Foundation

// Begrüßung
print("Willkommen zu meinem Swift-Programm!")

// Funktion definieren
func begruessung() {
    print("Das ist eine Beispiel-Funktion in Swift.")
}

// Funktion aufrufen
begruessung()

// Nutzereingabe
print("Bitte geben Sie Ihren Namen ein: ", terminator: "")
if let name = readLine() {
    // Begrüßung mit dem Namen
    print("Hallo, \(name)! Schön, dass Sie hier sind.")
}

// Programm beenden
print("Das Programm wurde beendet.")
